src/frontend/src/functions/auth.tsx

```diff
@@ -85,11 +86,18 @@ export const doTokenLogin = (token: string) => {
   const { fetchUserState } = useUserState.getState();
   const { fetchServerApiState } = useServerApiState.getState();
   const globalSettingsState = useGlobalSettingsState.getState();
+  const globalStatusState = useGlobalStatusState.getState();
   const userSettingsState = useUserSettingsState.getState();
 
+  // First, set the API token for auth
   setToken(token);
+
+  // Fetch user and server data
   fetchUserState();
   fetchServerApiState();
+
+  // Fetch settings
+  globalStatusState.fetchStatus();
   globalSettingsState.fetchSettings();
   userSettingsState.fetchSettings();
```

Comment:
I think this will run "in parallel" because there is no async code involved here, but the fetch functions use promises that are not awaited. I'm more wondering, why we can't put this state into the api state state?

Reply:
> why we can't put this state into the api state state?

Because each of these requests require auth - so I have pushed these off until we have confirmed that we are logged in.

Putting these requests into the general apiState manager is the cause for the "login dialog" popping up occasionally. This change also means that we do not get a whole heap of failed requests (due to no auth) in the developer console.

Reply:
Makes sense. That means we also move the status code request there too?

Reply:
Yep, already done. That's the "globalStatusState" manager